src/coreComponents/physicsSolvers/surfaceGeneration/EmbeddedSurfaceGenerator.cpp

```diff
@@ -128,7 +128,7 @@ void EmbeddedSurfaceGenerator::initializePostSubGroups()
   NewObjectLists newObjects;
 
   // Loop over all the fracture planes
-  geometricObjManager.forSubGroups< BoundedPlane >( [&]( BoundedPlane & fracture )
+  geometricObjManager.forSubGroups< BoundedPlanarObject >( [&]( BoundedPlanarObject & fracture )
```

Comment:
Since we are doing this we should probably change this to:

```suggestion
  geometricObjManager.forSubGroups< BoundedPlanarObject >( m_targetObjects  , [&]( BoundedPlanarObject & fracture )
```
where `m_targetObject` i

array1d<string> m_targetObject;